pillar/components/sspl.sls

```diff
@@ -96,12 +96,6 @@ sspl:
     threaded: "true"
     logging_interval: "3600"
   STORAGE_ENCLOSURE:
-    primary_controller_ip: "127.0.0.1"
```

Comment:
Am ok with this. But is this all? are we still referring from sspl.sls here, i remember we reduced use from sspl.sls earlier, but seems entries are lying around. Or sspl using rest of the stuff here still ?

Reply:
Ideally we should have only these entries in this file:
```
sspl:
  health_map_path: /opt/seagate/cortx/provisioner/generated_configs/healthmap/
  health_map_file: ees-schema.json
  rmq:
    user: rmq
    secret: "B4kf1T6Qso"
```

Reply:
Agree, it should be very minimal & i remember we shifted sspl config already, but stale entries remained here seems. @mariyappanp please confirm (check with @shubhamseagate too) and address

Reply:
@shubhamseagate patch is not having this change. There STORAGE_ENCLOSURE configs are still remained.

Reply:
Testing the same for cortx-1.0 on VM. I will post it once test done. BTW, I will test once in HW setup too.

Reply:
PR for cortx:
**pre-cortx-1.0**
https://github.com/Seagate/cortx-prvsnr/pull/591

Reply:
Ok you mean @shubhamseagate  patch taking care of the rest ? If yes lets move ahead with this change

Reply:
Yes

Reply:
Changed the ticket number as per the assignment. (EOS-13904 - **removing unused/duplicate kv pairs in consul**)